src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,13 @@ Pending
 
 * Update to use 2022-08-02-preview api version.
 
++++++++
+
+0.5.101
++++++++
+
+* Add --enable-vpa/--disable-vpa to enable/disable vertical pod autoscaler feature.
```

Comment:
Conflict? There's already 0.5.101 in [main](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/HISTORY.rst) branch. Also, if you want to release a new version, please also update `VERSION` in `setup.py`.

Reply:
Updated in the latest iteration